packages/venia-ui/lib/RootComponents/Search/categoryFilters.js

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from 'react';
+import { X as CloseIcon } from 'react-feather';
+import gql from 'graphql-tag';
+import { Query } from '@magento/venia-drivers';
+
+import { mergeClasses } from '../../classify';
+import Icon from '../../components/Icon';
+import getQueryParameterValue from '../../util/getQueryParameterValue';
+import defaultClasses from './categoryFilters.css';
+
+const GET_CATEGORY_NAME = gql`
```

Comment:
I know this is a small bit of code but should we store these with the other queries in the query dir?